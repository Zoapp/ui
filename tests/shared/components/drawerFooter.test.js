import React from "react";
import renderer from "react-test-renderer";
import DrawerFooter from "libs/components/drawerFooter";

describe("DrawerFooter", () => {
  it("should render", () => {
    const component = renderer.create(<DrawerFooter>test</DrawerFooter>);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
