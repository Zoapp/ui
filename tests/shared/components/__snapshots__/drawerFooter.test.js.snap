// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerFooter should render 1`] = `
<footer
  className="mdc-drawer__footer"
>
  <div
    className="mdc-drawer__footer-content"
  >
    test
  </div>
</footer>
`;
